src/Business/Detection/Common/Models/MCreateDetection.cs

```diff
@@ -0,0 +1,13 @@
+namespace Business.Detection.Common.Models
+{
+    using System;
+
+    public class MCreateDetection
+    {
+        public int Score { get; set; }
+
+        public string Class { get; set; }
+
+        public DateTime Timestamp { get; set; }
+    }
+}
```

Comment:
As far as we are gonna use this model for creation only, won't be better to move it under the `Detection/Creating/Models` folder?

Reply:
Done.